### DISCUSSION TITLE
Add a XT machine (Assistent128) to the 86Box

### DISCUSSION THREAD

**Discussion by usergithub64** [86Box/86Box#4010] [Ideas]
**Is your feature request related to a problem? Please describe.**
This is a request to add a XT machine (Assistent128) to the 86Box.

**Describe the solution you'd like**
Is it possible to add an Assistent128 machine to the 86Box?

**Additional context**
Perhaps this documentation will help.
Disassembled BIOS Assistent128:
[ASST128-BIOS_SOURCE.zip](https://github.com/86Box/86Box/files/8170808/ASST128-BIOS_SOURCE.zip)

Assistent128 ROM:
[asst128.zip](https://github.com/86Box/86Box/files/8283730/asst128.zip)
https://www.retrostic.com/roms/mame/assistent-128-52669

Assistent128 emulator:
https://yadi.sk/d/7r8nbtGn3Hf9DV

Information on Assistent128:
https://vk.com/club87371603

**Comment by ghost** [comment]
Translated Wikipedia article about **Assistant-128**: [Google Translate](https://ru-m-wikipedia-org.translate.goog/wiki/%D0%90%D1%81%D1%81%D0%B8%D1%81%D1%82%D0%B5%D0%BD%D1%82_(%D0%BA%D0%BE%D0%BC%D0%BF%D1%8C%D1%8E%D1%82%D0%B5%D1%80)?_x_tr_sl=ru&_x_tr_tl=en), [Yandex.Translate](https://translated.turbopages.org/proxy_u/ru-en.en.2493f3dc-62f53ba6-6ecaf000-74722d776562/https/ru.wikipedia.org/wiki/%D0%90%D1%81%D1%81%D0%B8%D1%81%D1%82%D0%B5%D0%BD%D1%82_(%D0%BA%D0%BE%D0%BC%D0%BF%D1%8C%D1%8E%D1%82%D0%B5%D1%80)).

Processed:
| Item | Value |
| :--- | ----- |
| CPU | KR1810VM86 (4.77 MHz), 16 bit — a Intel 8086 clone |
| RAM | 128 KiB, expandable up to 976 KiB (up to address `F3FF`) |
| ROM | 48 KiB "with **BASIC** flashed" |
| Video | CGA compatible (uses 16 KiB of RAM) |
| External Ports | - 1 parallel port;<br> - 2 joystick ports (guess: Gameport);<br>- 1 DIN port (guess: DIN 41524) for a tape recorder;<br> - RF (HF) and LF video output;<br>- **Color** for RGB, KSI (guess: Frame Sync Pulse), SSI (guess: Synchronous Serial Interface), "synchromix" (Composite Sync); |
| Additional information | "At the bottom of the system bus expansion, there's a slot to connect a Floppy Disk Drive, memory expansion unit and other devices." |
